src/main/java/com/palantir/docker/compose/DockerComposeRule.java

```diff
@@ -187,6 +175,18 @@ public Builder saveLogsTo(String path) {
             return logCollector(FileLogCollector.fromPath(path));
         }
 
+        /**
+         * @deprecated Please use <code>shutdownStrategy(ShutdownStrategy.SKIP)</code> instead.
+         */
+        @Deprecated
+        public Builder skipShutdown(boolean skipShutdown) {
```

Comment:
this means `skipShutdown(false)` does nothing? just want to make sure that's accurate